README.md

```diff
@@ -113,21 +113,31 @@ def auth():
     activity_details = client.get_activity_details(request.args['token'])
 ```
 
-## Profile Retrieval
+## Handling Users
 
-When your application receives a token via the exposed endpoint (it will be assigned to a query string parameter named `token`), you can easily retrieve the user profile by adding the following to your endpoint handler:
+When you retrieve the user profile, you receive a user ID generated by Yoti exclusively for your application.
+This means that if the same individual logs into another app, Yoti will assign her/him a different ID.
+You can use this ID to verify whether (for your application) the retrieved profile identifies a new or an existing user.
+Here is an example of how this works:
 
 ```python
-activity_details = client.get_activity_details(request.args['token'])
-user_profile = activity_details.user_profile
-    return render_template('profile.html',
-                           **user_profile)
-
+client = Client(YOTI_CLIENT_SDK_ID, YOTI_KEY_FILE_PATH)
+activity_details = client.get_activity_details(token)
+
+user_id = activity_details.user_profile.get('user_id')
```

Comment:
I think moving `activity_details.user_profile` into a separate object looks better:

`user_profile = activity_details.user_profile`

Reply:
Yes true, I'll change that